xray/commands/audit/generic/auditmanager.go

```diff
@@ -236,6 +243,7 @@ func doAudit(params *Params) (results []services.ScanResponse, isMultipleRoot bo
 			continue
 		}
 		techResults, e := audit.Audit(dependencyTrees, params.xrayGraphScanParams, params.serverDetails, params.progress, tech)
+		audit.BuildImpactPaths(techResults, params.dependencyTrees)
 		if e != nil {
```

Comment:
Let's put this error check right after the line that returns it.

Also, there's a missing error check at the bottom of this function. See these lines - 
```
	if errorList.Len() > 0 {
		err = errors.New(errorList.String())
	}
```